camel/societies/role_playing.py

```diff
@@ -120,8 +120,14 @@ def __init__(
             task_type=self.task_type, **(sys_msg_generator_kwargs or {}))
         (init_assistant_sys_msg, init_user_sys_msg,
          sys_msg_meta_dicts) = self.get_sys_message_info(
-             assistant_role_name, user_role_name, sys_msg_generator,
-             extend_sys_msg_meta_dicts)
+             assistant_role_name=assistant_role_name,
+             user_role_name=user_role_name, assistant_description=(
```

Comment:
Extract into `assistant_description` and `user_description` variable

Reply:
Thanks, done.